src/cfgparse-global.c

```diff
@@ -24,6 +24,11 @@
 #include <haproxy/peers.h>
 #include <haproxy/protocol.h>
 #include <haproxy/tools.h>
+#include <haproxy/proto_rhttp.h>
+
+extern struct protocol proto_rhttp;
```

Comment:
can you move it to `haproxy/proto_rhttp.h`, just under the `#include` instead please? So it would be similar to `proto_tcpv[46]` defined in `haproxy/proto_tcp.h`

Reply:
> can you move it to `haproxy/proto_rhttp.h`, just under the `#include` instead please? So it would be similar to `proto_tcpv[46]` defined in `haproxy/proto_tcp.h`

do not forget this ↑